libs/ui/src/lib/facets/facet-block/facet-block.component.ts

```diff
@@ -53,3 +53,13 @@ export class FacetBlockComponent implements OnInit {
     return this.model.more
   }
 }
+
+@Component({ selector: 'ui-facet-block', template: '' })
```

Comment:
I wouldn't put that here, why do you need this stub byt the way ?

Reply:
I use it to test the facet-list component without additional schemas. Should we just use the NO_ERRORS_SCHEMA everywhere ?

Reply:
It depends what you do want to test. But usually, when testing a component, we just test the inner behavior of the component, and don't dive deeper in the components tree.

Reply:
As explained here [https://angular.io/guide/testing-components-scenarios#nested-component-tests](url), the stub is only here to test the nested component interface, nothing deeper.